src/web/crate_details.rs

```diff
@@ -495,7 +496,7 @@ pub(crate) async fn get_all_releases(
                 .any(|s| Some(s) == params.target.as_ref())
         {
             inner_path.remove(0);
-            params.target.as_ref().unwrap()
+            &row.target_name
```

Comment:
this seems wrong, we still need the target in the links, right? 

( I see that the column name `target_name` is confusing, since it's actually the package-name)

Reply:
It was indeed wrong. The problem came from two places: the test that didn't add a target and thus this modification which came from my misunderstanding.